### DISCUSSION TITLE
Où changer "untiteld" par un tire voulu à la création ?

### DISCUSSION THREAD

**Discussion by LauwCost** [senshu/Sozi#703] [Q&A]
Bonjour, 

Je souhaite remplacer le "untitled" par défaut qui apparaît dans le titr edes onglet de pages générées par Sozi mais je n'ai pas trouvé où c'était dans l'interface.

Merci pour votre aide.

Librement, 
Lauw'

**Comment by ejvindh** [comment]
I think you're right that it isn't part of the current userinterface. I don't know if it is in the development-pipe, but until it is, a hack-around is to open the generated html-file with a text-editor (like notepad if you're in windows), search for the string "<title id" and replace the string "Untitled" that is placed between the tags <title></title> according to your wishes.

**Reply by LauwCost** [reply]
Thank you for your reply.
Yes, that's how I did it.
I was hoping for a feature that I'd overlooked, but it doesn't seem to be the case.
Thanks again for your reply.

**Comment by aumouvantsillage** [comment]
Le titre de la présentation est extrait automatiquement des propriétés du document SVG.
Si vous utilisez Inkscape, vous pouvez ajouter un titre dans les métadonnées du document : menu "Fichier", Propriétés du document, onglet "Métadonnées".

N.B. Voir également la conversation #550
